database/seeders/DatabaseSeeder.php

```diff
@@ -12,6 +12,10 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
+
+        $this->call(RoleSeeder::class);
+        $this->call(PermissionSeeder::class);
+
         \App\Models\User::factory(10)->create();
```

Comment:
You can update the factory to generate members by default or even create model factory states for super admin and member and call them here.